test/integration/components/listeners.test.ts

```diff
@@ -29,11 +25,12 @@ describe('#listeners', () => {
       useRequestId: false,
       enableEventEngine: true,
       autoNetworkDetection: false,
+      keepAlive: false,
```

Comment:
any reason or side effect here ?

Reply:
@mohitpubnub removed it. Added when debugged why test was freezing after printing out completion results.